### PR TITLE
Update newline

### PR DESCRIPTION
**Pull request by Prithivi-Raj**
ewm eq

### CODE REVIEW

**Comment by niharikachavati** [comment]
include berries

**Review by Prithivi-Raj — COMMENTED** [review]
maMXKAMKAM

File: newline [hunk 1]
```diff
@@ -1 +1,3 @@
 helloo 
+qaxmqmxq
```
Comment (niharikachavati) [inline newline:2]: good

File: newline [hunk 2]
```diff
@@ -1 +1,3 @@
...
+beeryyy
```
Comment (niharikachavati) [inline newline:3]: nice

**Comment by Prithivi-Raj** [comment]
closing

**Pull request closed without merging**